doc/user_manual_en.md

```diff
@@ -0,0 +1,111 @@
+# qTox User Manual
+## Index
+
+* [Settings](#Settings)
+* [Groupchats](#Groupchats)
+* [Multi Window Mode](#multi-window-mode)
+* [Keyboard Shortcuts](#keyboard-shortcuts)
+
+## Settings
+### General
+
+#### General Settings
+* _Language:_ Changes which language the qTox interface uses.
+* _Autostart:_ If set, qTox will start when you login on your computer.
```

Comment:
Will autostart with the profile with which user ticked on `Autostart`, but only if profile isn't encrypted (doesn't have the password), otherwise it'll just start qTox, but without starting profile ← that probably should be mentioned, since quite a few users were a bit confused why autostart doesn't seem to "work".